dotenv-sample

```diff
@@ -1,13 +1,12 @@
 # The name of this backend
 OPENSAFELY_BACKEND=tpp
 
-# Credentials for logging into the job server
-OPENSAFELY_QUEUE_USER=user
-OPENSAFELY_QUEUE_PASS=pass
-
 # The endpoint to poll for jobs
 OPENSAFELY_JOB_SERVER_ENDPOINT=https://jobs.opensafely.org/jobs/
 
+# Credentials for logging into the job server
+export JOB_SERVER_TOKEN=pass
```

Comment:
I think you want:
```
OPENSAFELY_JOB_SERVER_TOKEN=pass
```
(without the `export`)

Reply:
ooh yes, definitely copy/pasta from my own `.envrc`!